metricbeat/module/couchdb/server/_meta/fields.yml

```diff
@@ -0,0 +1,801 @@
+            - name: mean
+              type: scaled_float
+              description: >
+            - name: stddev    
+              type: scaled_float
+              description: >     
+            - name: min
+              type: scaled_float          
+              description: >
+            - name: max                    
+              type: scaled_float
+              description: >
+        - name: DELETE
+          type: group
+          description: >       
+          fields:
+            - name: description
+              type: keyword  
+              description: >
+            - name: current          
+              type: scaled_float
+              description: >             
+            - name: sum
+              type: scaled_float                    
+              description: >
+            - name: mean
+              type: scaled_float
+              description: >
+            - name: stddev    
+              type: scaled_float
+              description: >     
+            - name: min
+              type: scaled_float          
+              description: >
+            - name: max                    
+              type: scaled_float
+              description: >
+        - name: GET
+          type: group
+          description: >                  
+          fields:
+            - name: description
+              type: keyword  
+              description: >
+            - name: current          
+              type: scaled_float
+              description: >             
+            - name: sum
+              type: scaled_float                    
+              description: >
+            - name: mean
+              type: scaled_float
+              description: >
+            - name: stddev    
+              type: scaled_float
+              description: >     
+            - name: min
+              type: scaled_float          
+              description: >
+            - name: max                    
+              type: scaled_float
+              description: >
+        - name: PUT
+          type: group
+          description: >                  
+          fields:
+            - name: description
+              type: keyword  
+              description: >
+            - name: current          
+              type: scaled_float
+              description: >             
+            - name: sum
+              type: scaled_float                    
+              description: >
+            - name: mean
+              type: scaled_float
+              description: >
+            - name: stddev    
+              type: scaled_float
+              description: >     
+            - name: min
+              type: scaled_float          
+              description: >
+            - name: max                    
+              type: scaled_float
+              description: >
+    - name: httpd_status_codes
+      type: group
+      description: >
+        Example field
+      fields:
+        - name: num200
```

Comment:
I think fields here should be named just with the number i.e. `200` instead of `num200`.